bazaarvoice/astyanax
# PD-211832 : Astyanax_Jackson_Core_update_to_2.14.2

**Jira Issue**
https://bazaarvoice.atlassian.net/browse/PD-211832

**What Are We Doing Here?**
SNYK Vulnerability Remediation for Astyanax Jackson Library.
--> Update Jackson fasterxml core (core, annotations, databind) to 2.14.2.

**How to Test and Verify**

Check out local build :
Unit testing : mvn clean install
<img width="1728" alt="Screenshot 2023-06-26 at 3 46 21 PM" src="https://github.com/bazaarvoice/astyanax/assets/134580247/40bf4ac3-8327-4c8d-bf03-60a7aebb6276">


**Risk**
Level : Low

**Required Testing**
Manual.

**Risk Summary**
Since risk level will be low,we will be monitoring for vulnerability issues of jackson library in snyk dashboard.
